onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/impl/CheckForPlausibleStopTimes.java

```diff
@@ -49,15 +51,39 @@ public void run(TransformContext context, GtfsMutableRelationalDao dao) {
                     oldTime = newTime;
                 }
                 //check if the bus takes more than five hours between stops
-                if(newTime.getArrivalTime() - oldTime.getDepartureTime() > 60*60*5){
-                    _log.error("Trip {} on Route {} is scheduled for unrealistic transit time between {} at {}, and {} at {}", trip.getId(), trip.getRoute(), oldTime.getId(), oldTime.getDepartureTime(), newTime.getId(), newTime.getDepartureTime());
-                    //es.publishMessage(getTopic(), "Trip " + trip.getId() + " on Route "+ trip.getRoute() +" is scheduled for unrealistic transit time between " + oldTime.getId()+ " at " + oldTime.getDepartureTime() + ", and " +  newTime.getId() + " at " + newTime.getDepartureTime()););
+                if(newTime.getArrivalTime() - oldTime.getDepartureTime() > 5 * MINUTES_PER_HOUR * SECONDS_PER_MINUTE){
+                    _log.error("Trip {} on Route {} is scheduled for unrealistic transit time between trip {} at {}, and trip {} at {}", trip.getId(), trip.getRoute(), oldTime.getId(), humanReadableTime(oldTime.getDepartureTime()), newTime.getId(), humanReadableTime(newTime.getArrivalTime()));
+                    es.publishMessage(getTopic(), "Trip " + trip.getId() + " on Route "+ trip.getRoute() +" is scheduled for unrealistic transit time when traveling between stoptime" + oldTime.getId()+ " at " + oldTime.getDepartureTime() + ", and stoptime" +  newTime.getId() + " at " + newTime.getDepartureTime());
+
+                    _log.error("Trip "+trip.getId().getId()+" on Route "+trip.getRoute().getId().getId()+" is scheduled for unrealistic transit time between trip "+ oldTime.getId()+" at "+humanReadableTime(oldTime.getDepartureTime())+", and trip "+newTime.getId()+" at " + humanReadableTime(newTime.getArrivalTime()));
+
                 }
                 oldTime= newTime;
             }
+        }
+    }
 
+    private String humanReadableTime(int time){
+        String output;
+        int hours;
+        int minutes;
+        int seconds;
+        hours = time/(MINUTES_PER_HOUR*SECONDS_PER_MINUTE);
+        minutes = (time - hours*MINUTES_PER_HOUR*SECONDS_PER_MINUTE)/SECONDS_PER_MINUTE;
+        seconds = (time - hours*MINUTES_PER_HOUR*SECONDS_PER_MINUTE - minutes*SECONDS_PER_MINUTE);
+        output = intTimeToString(hours) + ":" + intTimeToString(minutes) + ":" + intTimeToString(seconds);
+        return output;
+    }
+
+    private String intTimeToString(int time){
```

Comment:
This is called a "left pad".  Again, libraries exist to do this, but the date format comment above makes this completely unnecessary

Reply:
Thank you for the name of that!